src/tvdb/client.py

```diff
@@ -49,6 +50,9 @@ def parse_media_id(media_id: int | str) -> int:
         return media_id
 
 
+cache = SimpleMemoryCache()
```

Comment:
It'd be nicer to have a single global cache stored in the `Bot` class, that is then passed to `Client` through the constructor as an argument.

Because this will be a global cache, you'll also want to use `namespace` to isolate the keys used here, maybe something like `tvdb-media` could work well.